client/rpc/src/state/tests.rs

```diff
@@ -402,7 +402,7 @@ fn should_return_runtime_version() {
 
 	let result = "{\"specName\":\"test\",\"implName\":\"parity-test\",\"authoringVersion\":1,\
 		\"specVersion\":1,\"implVersion\":2,\"apis\":[[\"0xdf6acb689907609b\",2],\
-		[\"0x37e397fc7c91f5e4\",1],[\"0xd2bc9897eed08f15\",1],[\"0x40fe3ad401f8959a\",5],\
+		[\"0x37e397fc7c91f5e4\",1],[\"0xd2bc9897eed08f15\",1],[\"0x40fe3ad401f8959a\",4],\
```

Comment:
Wouldn't this revert cause issues? Or do we feel believe it wasn't deployed yet? I'd be safe and just bump the version instead of reverting it.

Reply:
It wasn't deployed on Kusama yet and we are still in alpha. This is okay.